docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,20 @@
 Pulp Puppet 2.8 Release Notes
 =============================
 
+Pulp Puppet 2.8.4
+=================
+
+- A migration included in this release fixes an issue that was caused by incorrect Puppet module's
```

Comment:
s/that was caused by incorrect Puppet module's name interpretation/which caused some Puppet modules to not be installable/.